src/Entity/Fraction.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mathematicator\Numbers\Entity;
+
+use Brick\Math\BigNumber;
+use Stringable;
+
+/**
+ * Entity to store simple and compound fractions
+ */
+class Fraction
```

Comment:
Fraction should be subtype of `SmartNumber` which can represent fraction too.

Reply:
I think that number should be number and fraction should be fraction, but SmartNumber can use it internally.